go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/binance-chain/tss-lib v1.3.1
 	github.com/btcsuite/btcd v0.20.1-beta
+	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
```

Comment:
I see that we have:
```
replace (
	github.com/BurntSushi/toml => github.com/keep-network/toml v0.3.0
	github.com/blockcypher/gobcy => github.com/keep-network/gobcy v1.3.1
	github.com/btcsuite/btcd => github.com/keep-network/btcd v0.0.0-20190427004231-96897255fd17
	github.com/btcsuite/btcutil => github.com/keep-network/btcutil v0.0.0-20190425235716-9e5f4b9a998d
```

Does that mean I don't need the above line? It looks like my environment generated this change

Reply:
I… Don't know, but it feels right to have this in there.